### PR TITLE
March 2024 iska's body markings

### PR DESCRIPTION
**Pull request by iskawhiskers**
## About The Pull Request

Adds the following body marking sprites:
weight2_chest_m
weight2_chest_f
blankface2_head
bigoleyes_head
monster2_head
monster3_head

## How This Contributes To The Nova Sector Roleplay Experience

Sprites good

## Proof of Testing

Image of character with monster3_head used
Using monster3_head sprite
Tested on Bubberstation, but should work here too

## Changelog

:cl:
image: Added body marking sprites (monster2, monster3, weight2 (masc/fem variants), blankface2, bigoleyes)
/:cl:

### CODE REVIEW

File: modular_nova/modules/customization/modules/mob/dead/new_player/body_markings/body_markings.dm [hunk 1]
```diff
@@ -79,12 +79,31 @@
 	default_color = "#CCCCCC"
 	affected_bodyparts = HEAD
 
+/datum/body_marking/other/blank_face2
+	name = "Blank round face, Alt"
```
Comment (GoldenAlpharex) [inline modular_nova/modules/customization/modules/mob/dead/new_player/body_markings/body_markings.dm:83]:
```suggestion
	name = "Blank Round Face, Alt"
```

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]
Could you include more screenshots featuring each marking individually? It's kinda hard to tell what's meant to be what.

**Comment by iskawhiskers** [comment]
> Could you include more screenshots featuring each marking individually? It's kinda hard to tell what's meant to be what.

Not without compiling the entirety of the source code myself and that will take a ton of time with my terrible internet.  weight2 is just a greyscale version of the body weight body marking and monster2 is a recolourable and slightly better version of the monster mouth marking, though

**Comment by Nerev4r** [comment]
image
grayscale bodyweight on an amab body

image
on an afab body

image
alt blank round face
image


image
as god intended
image


image
monster mouth included
image


image
eye compatible version
image

image
quit staring at me with them big ol eyes
image

**Comment by Nerev4r** [comment]
image
image
me when i

**Comment by iskawhiskers** [comment]
Thank you based Juniper

**Review by GoldenAlpharex — APPROVED** [review]
Yeah, this seems fine to me, even if I know that characters without eyes will probably look weird with stuff likes glasses and the likes, but that's not like it's not already an issue with xenomorph hybrids and the likes.

**Pull request merged**